Use new team drives arg instead of deprecated one

ali-tny opened the pull request:
The calling signature for using this argument when creating files was also incorrect, so I update it to be a argument of the actual method rather than being in the args variable. Previously, trying to write to shared drives failed.

ali-tny committed:
Use new team drives arg instead of deprecated one

Also the calling signature for using this argument when creating files
was incorrect, so I update it to be a argument of the actual method
rather than being in the args variable.

j0nnyr0berts reviewed (approved):
Very nice. Would you mind bumping the version + releasing?

ali-tny committed:
Correct version number in CHANGELOG

Also make formatting of bullet points consistent across the file

ali-tny committed:
Add CHANGELOG links and "Unreleased" section

To follow the keep a change log (https://keepachangelog.com/en/1.0.0/)
format properly, including using the recommended markdown format. Links
provide users a place to see all the changes from one release to
another, and the unreleased section means that each PR can update the
"Unreleased" section, and then keeping the changelog up to date at
release time is super easy (just turn "Unreleased" to the version name
and make a new empty unreleased section).

ali-tny committed:
Prepare 0.0.3 release
